### PR TITLE
Implement --from-clipboard option

### PR DESCRIPTION
**Pull request by sathwikmatsa**
Added --from-clipboard option to numconverter.
Resolves issue #12

### CODE REVIEW

**Comment by ZNielsen** [comment]
I think we are going to hold off on this ticket until we get #8 sorted out.  @Jonathas-Conceicao has a solution in the works that will make this feature a little less straight-forward, but much more compatible with cross platform operation.

If you are still interested in implementing this feature after that code is merged, I can assign #12 to you at that time.

We appreciate your contribution!

**Comment by ZNielsen** [comment]
#22 is now merged in and changes how this would be implemented.  Would you still like to work on this?  If yes, I'll assign the issue to you.  Thanks!

**Comment by sathwikmatsa** [comment]
Yes, I would like to work on this.

File: README.md
Comment (ZNielsen) [inline README.md:78]: Not really a standard, but I like to keep required arguments in `<>` brackets, optional arguments in `[]` brackets.

https://stackoverflow.com/a/36787811/8862937

Comment (ZNielsen) [reply README.md:78]: Whoops, obviously that argument is now optional due to the `--from-clipboard` option.   Sorry, commented before thinking about it!

**Review by ZNielsen — APPROVED** [review]

**Pull request merged**
